adafruit_dash_display.py

```diff
@@ -125,18 +125,23 @@ def last_val(self, value: str):
 
 
 class Hub:  # pylint: disable=too-many-instance-attributes
-    """Object that lets you make an IOT dashboard"""
+    """
+    Object that lets you make an IOT dashboard
+
+    :param display: The display object.
+    :param io: The io object.
+    :param nav: The navigation buttons.
+    """
 
-    # pylint: disable=invalid-name
     def __init__(
         self,
         display: displayio.Display,
-        io: IO_MQTT,
+        io_object: IO_MQTT,
         nav: Tuple[DigitalInOut, ...],
     ):
         self.display = display
 
-        self.io = io  # pylint: disable=invalid-name
+        self.io_object = io_object
```

Comment:
```suggestion
        self.io_mqtt = io_mqtt
```
and change the `self` attribute also

Reply:
Got it!